package.json

```diff
@@ -2,7 +2,7 @@
   "name": "react-vis",
   "version": "0.9.0",
   "license": "MIT",
-  "author": "Anton Bulyenov <antonb@uber.com>",
+  "author": "Anton Bulyenov <visualization@uber.com>",
```

Comment:
Should probably updated to someone who is not anton, as he isn't a maintainer anymore